src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(
+    compareFunction = function(a, b) {
+      return String(a) < String(b) ? -1
+        : (String(a) > String(b) ? 1 : 0);
+    }
```

Comment:
try to create separate function for default `compareFunction` 
and do not use 2 ternary operators in a row because it adds complexity to read